components/Typography.js

```diff
@@ -0,0 +1,63 @@
+import styled, { css } from "styled-components";
+
+const SetBackground = css`
+  ${(p) => p.background && `background: ${p.background}`};
+`;
+
+const SetColor = css`
+  ${(p) => p.color && `color: ${p.color}`};
+`;
+
+export const LargeTitle = styled.h1.attrs((p) => ({
```

Comment:
I named the typography components just as how they're named on figma:
- `Title1` is actually its own heading (used for section headings), different from `LargeTitle`
<img width="1027" alt="Screen Shot 2021-08-15 at 7 44 44 PM" src="https://user-images.githubusercontent.com/38872354/129504644-693b8a9d-43d9-464a-b51b-8540a3c8138f.png">
<img width="852" alt="Screen Shot 2021-08-15 at 7 45 33 PM" src="https://user-images.githubusercontent.com/38872354/129504704-fa9d1470-4bf9-4ea7-82d7-2ddf4ab17cdc.png">

- The hero section only uses `LargeTitle` and `Title2`. There's a lot of typography components, and since not all of them were needed for my change in this PR, I thought the rest were out of scope :p . I've opened #48 for the remaining typography components (there's 4/7 left)
<img width="863" alt="Screen Shot 2021-08-15 at 7 45 46 PM" src="https://user-images.githubusercontent.com/38872354/129504726-fbe248ab-2320-47d5-bb90-42201a172f81.png">

Reply:
Ohhh I see, yeah totally makes sense then! Sorry didn't see that, thanks for clarifying!